src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -29,6 +29,7 @@ interface ChatboxProps {
   isVenueChat?: boolean;
 }
 
+// TODO: we have a ChatBox in organisms but also in molecules.. are they the same? Can we de-dupe them?
```

Comment:
Yep we should look into this
```suggestion
// @debt TODO: we have a ChatBox in organisms but also in molecules.. are they the same? Can we de-dupe them?
```